src/Runner.Worker/ExecutionContext.cs

```diff
@@ -91,6 +93,7 @@ public interface IExecutionContext : IRunnerService
         void SetOutput(string name, string value, out string reference);
         void SetTimeout(TimeSpan? timeout);
         void AddIssue(Issue issue, string message = null);
+        void AddIssueToTimelineRecord(Issue issue);
```

Comment:
it's kind of weird to have 2 `AddIssue()` methods and hard for the caller to figure out which one they should use.